tests/testArbitrary.ml

```diff
@@ -24,6 +24,12 @@ let arb_positive_lqt = QCheck.map (fun x -> lqt_of_denomination (Ligo.nat_from_l
 
 let arb_nat = QCheck.map (fun x -> Ligo.nat_from_literal ((string_of_int x) ^ "n")) QCheck.(0 -- max_int)
 
+(* somewhere between 0 and 3 *)
+let arb_small_nat =
+  QCheck.map
+    (fun x -> Ligo.nat_from_literal ((string_of_int x) ^ "n"))
+    QCheck.(1 -- ((max_int / 479_988_656_967) / 4))
```

Comment:
Ah, I have no idea, I just made `arb_small_nat` as a copy of `arb_small_tez`, which was already convoluted (for no reason that I can see). I'll clean them both up :+1:

Reply:
I actually don't know anything more about this either :sweat_smile:. As far as I know there is nothing special about this number. In my case it looks like I've used this value for adding a random delta to a given amount of tez, but as far as I know slightly increasing or decreasing this value should make no difference in that context.

Reply:
Hahaha, nice. Now that I'm working on #192 I'll switch it to `10_000_000_000` then. I've had to change many of our `max_int` likewise anyway, since #192 uses a bounded representation for tez and otherwise tests would overflow on addition.